monitoring/perf_context.cc

```diff
@@ -186,8 +205,29 @@ std::string PerfContext::ToString(bool exclude_zero_counters) const {
   PERF_CONTEXT_OUTPUT(env_lock_file_nanos);
   PERF_CONTEXT_OUTPUT(env_unlock_file_nanos);
   PERF_CONTEXT_OUTPUT(env_new_logger_nanos);
+  PERF_CONTEXT_BY_LEVEL_OUTPUT_ONE_COUNTER(bloom_filter_useful);
+  PERF_CONTEXT_BY_LEVEL_OUTPUT_ONE_COUNTER(bloom_filter_full_positive);
+  PERF_CONTEXT_BY_LEVEL_OUTPUT_ONE_COUNTER(bloom_filter_full_true_positive);
```

Comment:
added `PerfContextByLevel::ToString` to print all counters at a particular level, user can call by `(*(get_perf_context()->level_to_perf_context))[level].ToString(true);`

Reply:
I would avoid to add one extra ToString() unless we are convinced that it is needed. My initial question is that, how did you choose between this format:
```
bloom_filter_useful: 5@0, 4@1, 0@2
bloom_filter_full_positive: 0@0, 0:1, 0:2
```
and
```
Level 0:
  bloom_filter_useful: 5
  bloom_filter_full_positive: 0
Level 1:
  bloom_filter_useful: 4
  bloom_filter_full_positive: 0
Level 2:
  bloom_filter_useful: 0
  bloom_filter_full_positive: 0
```
I'm OK with either way. I'm just curious on how it was decided.

Reply:
I chose the former as default since I think in most cases we want to know the distribution of a **particular counter** at different levels, but also added ToString() to support the latter because the latter might also be useful when user want to browse through all counters and get a bigger picture

Reply:
I suggest we remove this ToString() for now. I'm not sure that people will use it.